cardano-cli/src/Cardano/CLI/Shelley/Parsers.hs

```diff
@@ -1879,29 +1879,35 @@ pProtocol =
 
     pEpochSlots :: Parser EpochSlots
     pEpochSlots =
-      EpochSlots <$>
-        ( Opt.option Opt.auto
-            (  Opt.long "epoch-slots"
-            <> Opt.metavar "NATURAL"
-            <> Opt.help "The number of slots per epoch (default is 21600)."
-            )
-        <|>
-          -- Default to the mainnet value.
-          pure 21600
+        ( EpochSlots <$>
+            Opt.option Opt.auto
+              (  Opt.long "epoch-slots"
+              <> Opt.metavar "NATURAL"
+              <> Opt.help "The number of slots per epoch (default is 21600)."
```

Comment:
```suggestion
              <> Opt.help "The number of slots per epoch for the Byron era (default is 21600)."
```
since it's different for Shelley.